control-with-full-hash/README.md

```diff
@@ -0,0 +1,100 @@
+# Control with URL
+
+In this example the user can search in either datacube or dashboard view to update an iframe with a new Imply-UI URL that applies a page filter. In datacube view the user also has the option of selecting a dimension from a dropdown menu to generate new URL and update the iframe. 
+
+![screenshot of control-with-url-example](images/data_cube.png "control-with-url-example")
+![screenshot of control-with-url-example](images/dashboard.png "control-with-url-example")
+
+To run: 
+
+`npm install`
+
+`npm start`
```

Comment:
Why do we need npm install for this example?

Reply:
I left it because of express. Should we not be using express entirely anymore ?